<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <author>MahirP</author>
    <class>mainWindow</class>
    <widget class="QMainWindow" name="mainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1280</width>
                <height>750</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="autoFillBackground">
            <bool>false</bool>
        </property>
        <property name="styleSheet">
            <string notr="true">QWidget{
                color:#FFF;
                background-color: rgb(20,20,20);
                }
                QWidget:disabled {
                color:#757575;
                }
                QLineEdit,QSlider, QScrollArea, QProgressBar,QSpinBox,QDoubleSpinBox{
                border:1px solid rgb(40,40,40);
                }
                QLineEdit:focus{
                border: 1px solid rgb(14, 77, 179);
                }
                QCheckBox::indicator{
                border:2px solid rgb(40,40,40);
                }
                QCheckBox::indicator::checked{
                color: rgb(14, 77, 179);
                background:rgb(14, 77, 179);
                }
                QRadioButton::indicator{
                border:2px solid rgb(40,40,40);
                width:10px;
                height:10px;
                border-radius:5px;
                background:None;
                }
                QRadioButton::indicator::checked{
                color: rgb(14, 77, 179);
                background:rgb(14, 77, 179);
                }
                QScrollBar:vertical{
                width:6px;
                }
                QScrollBar:horizontal{
                height:6px;
                }
                QScrollBar::handle {
                margin:1px;
                border:1px solid black;
                border-radius: 10px;
                background-color: rgb(14, 77, 179);
                width:5px;
                }
                QScrollBar::add-line {
                height: 0px;
                subcontrol-position: bottom;
                subcontrol-origin: margin;
                }
                QScrollBar::sub-line {
                height: 0 px;
                subcontrol-position: top;
                subcontrol-origin: margin;
                }
            </string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <property name="minimumSize">
                <size>
                    <width>1280</width>
                    <height>720</height>
                </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_3" columnstretch="5">
                <property name="leftMargin">
                    <number>0</number>
                </property>
                <property name="topMargin">
                    <number>0</number>
                </property>
                <property name="rightMargin">
                    <number>0</number>
                </property>
                <property name="bottomMargin">
                    <number>0</number>
                </property>
                <property name="spacing">
                    <number>0</number>
                </property>
                <item row="1" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
                        <item>
                            <widget class="QTabWidget" name="viewTabWidget">
                                <property name="styleSheet">
                                    <string notr="true">QTabWidget,QTabBar{
                                        color:rgb(86, 152, 198);
                                        }
                                    </string>
                                </property>
                                <property name="currentIndex">
                                    <number>-1</number>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="status">
                                <property name="text">
                                    <string/>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1280</width>
                    <height>30</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuProject">
                <property name="title">
                    <string>Project</string>
                </property>
                <addaction name="actionNew"/>
                <addaction name="actionLoad"/>
                <addaction name="actionSave_Config"/>
            </widget>
            <widget class="QMenu" name="menuData_Files">
                <property name="title">
                    <string>Data Files</string>
                </property>
                <addaction name="actionSave"/>
                <addaction name="actionReload_Files"/>
            </widget>
            <widget class="QMenu" name="menuTools">
                <property name="title">
                    <string>Tools</string>
                </property>
                <widget class="QMenu" name="menuCalibration">
                    <property name="enabled">
                        <bool>true</bool>
                    </property>
                    <property name="title">
                        <string>Calibration</string>
                    </property>
                    <addaction name="actionCalibration"/>
                    <addaction name="actionImport_DLT_Coefficients"/>
                </widget>
                <addaction name="menuCalibration"/>
            </widget>
            <addaction name="menuProject"/>
            <addaction name="menuData_Files"/>
            <addaction name="menuTools"/>
        </widget>
        <action name="actionNew">
            <property name="text">
                <string>New</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+N</string>
            </property>
        </action>
        <action name="actionLoad">
            <property name="text">
                <string>Open</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+O</string>
            </property>
        </action>
        <action name="actionSave">
            <property name="text">
                <string>Save Data</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+S</string>
            </property>
        </action>
        <action name="actionSave_Config">
            <property name="text">
                <string>Save Config</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Alt+S</string>
            </property>
        </action>
        <action name="actionReload_Files">
            <property name="text">
                <string>Reload Data</string>
            </property>
        </action>
        <action name="actionCalibration">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="text">
                <string>Calibration Toolbox</string>
            </property>
        </action>
        <action name="actionImport_DLT_Coefficients">
            <property name="text">
                <string>Import DLT Coefficients</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
